oci/private/pull.bzl

```diff
@@ -490,9 +490,7 @@ buildozer 'set digest "sha256:{digest}"' 'remove tag' 'remove platforms' {option
             name = rctx.attr.name,
             target_name = rctx.attr.target_name,
             platform_map = {
```

Comment:
Didn't know this was even possible - you might want to use `repr({...})` as I don't know whether `str({...})` is guaranteed to roundtrip.

Reply:
either way, you have to pass the labels through `str()` -- `repr()` doesn't roundtrip labels, unfortunately...

Reply:
This seems to work now. THanks!